airqo-platform/AirQo-api
# addresses issue #1075

**_WHAT DOES THIS PR DO?_**
- Closes https://github.com/airqo-platform/AirQo-api/issues/1075

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-170]
- GitHub issues
    - Closes https://github.com/airqo-platform/AirQo-api/issues/1075

**_HOW DO I TEST OUT THIS PR?_**
```
cd src/auth-service
npm run stage-mac
```

🚨👇 Please ensure that your `.env` has values for the following variable: 🚨👇

- _MONGO_STAGE_URI_

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [request access](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-053764bb-d840-4cf8-88d2-ffbb8a347247)




[AN-170]: https://airqoteam.atlassian.net/browse/AN-170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
